hat/assets/js/apps/Iaso/domains/app/translations/fr.json

```
@@ -390,8 +391,10 @@
     "iaso.instance.missingFile": "Aucun fichier trouvé",
     "iaso.instance.missingGeolocation": "Aucune soumission trouvée avec une localisation",
     "iaso.instance.multipleInstancesOneOrgUnitWarningMessage": "Plusieurs soumissions utilisent la même unité d'org.",
+    "iaso.instance.multipleReferenceInstancesOneOrgUnitWarningMessage": "Plusieurs soumissions de référence utilisent les mêmes unités d'org. et les mêmes formulaires",
     "iaso.instance.noGpsForSomeInstaces": "Certaines soumissions n'ont pas de GPS. Rien ne sera appliqué pour ces unités d'organisation.",
     "iaso.instance.NoLocksHistory": "Il n'y a pas d'historique",
+    "iaso.instance.noReferenceSubmissionsWarningMessage": "{selectedSubmissionsCount} soumissions sélectionnées ne sont pas des soumissions de référence",
```

So that it matches the English text:
```suggestion
    "iaso.instance.noReferenceSubmissionsWarningMessage": "{selectedSubmissionsCount} soumissions sélectionnées ne peuvent pas devenir des soumissions de référence (leur formulaire n'est pas configuré comme étant un formulaire de référence pour leur type d'unité d'org. respectif)",
```